lib/Doctrine/ORM/Configuration.php

```diff
@@ -162,6 +164,14 @@ public function setMetadataDriverImpl(MappingDriver $driverImpl)
      */
     public function newDefaultAnnotationDriver($paths = [], $useSimpleAnnotationReader = true)
     {
+        if (! class_exists(AnnotationReader::class)) {
+            throw new LogicException(sprintf(
+                'The annotation metadata driver cannot be enabled because the "doctrine/annotations" library'
+                . ' is not installed. Please run "composer require doctrine/annotations" or chose a different'
```

Comment:
"choose, chose, chosen" :slightly_smiling_face: 

```suggestion
                . ' is not installed. Please run "composer require doctrine/annotations" or choose a different'
```